api/admin.go

```diff
@@ -334,9 +346,61 @@ func (a *API) adminUserDelete(w http.ResponseWriter, r *http.Request) error {
 			return internalServerError("Error recording audit log entry").WithInternalError(terr)
 		}
 
-		if terr := tx.Destroy(user); terr != nil {
-			return internalServerError("Database error deleting user").WithInternalError(terr)
+		if params.IsSoftDelete == "true" {
```

Comment:
URI spec doesn't really say [what counts as a boolean](https://stackoverflow.com/questions/6461668/boolean-in-a-uri-query). Guess we can just define it in the docs

Reply:
made the `is_soft_delete` query param case-insensitive by converting it to lowercase so this should be handled